cvat/settings/base.py

```diff
@@ -177,7 +177,7 @@ def add_ssh_keys():
         'rest_framework.throttling.AnonRateThrottle',
     ],
     'DEFAULT_THROTTLE_RATES': {
-        'anon': '100/minute',
+        'anon': '150/minute',
```

Comment:
I don't think this change should be done here. @azhavoro, please comment.

Reply:
I revert this change, and now the only one test failed: https://github.com/opencv/cvat/actions/runs/3225925269/jobs/5279024065.

I would like to merge this PR and fix this problem in another PR